install_config/topics/ocp_osp_prerequisites.adoc

```diff
@@ -7,22 +7,21 @@ regarding the prerequisites and configuration changes required for an {product-t
 OpenStack environment are discussed in detail.
 
 NOTE: All the OpenStack CLI commands in this reference environment are executed
-using the CLI `openstack` commands within the OpenStack director node. If using a
-workstation or laptop to execute these commands instead of the OpenStack
-director node, please ensure to install the following packages found
-within the specified repositories.
+using the CLI `openstack` commands from a different node than the OpenStack
+director node because of package conflicts for newer versions of Ansible.
```

Comment:
Are we able to be more specific about version conflicts, here? "Newer" is okay if not.

Reply:
Yes, it's Ansible >= 2.6. I'll change it.